content/redirects.md

```diff
@@ -13,5 +13,5 @@ permalink: /{{ redirect.from }}
 
 {% set to = collections.all | getPage(redirect.to) %}
 
-Redirecting you to [{{ post.name(to) }}]({{ to.url }}).
+Redirecting you to [{{ post.name(to) if to else redirect.to }}]({{ redirect.to }}).
```

Comment:
For reasons that aren't clear to me, at the time of this template compilation it seems that most of the `Tag` pages aren't yet part of `collections.all`. This is hardly visible in practice, but adds a fallback to display the URL if the page title isn't yet available.